source/common/res/features/budget-daily-spending/main.js

```diff
@@ -21,11 +21,24 @@
         },
 
         observe(changedNodes) {
-          if (changedNodes.has('budget-inspector')) {
+          if (changedNodes.has('navlink-budget active') ||
+              changedNodes.has('budget-table-cell-available-div user-data') ||
+              changedNodes.has('budget-inspector')) {
             ynabToolKit.dailySpending.invoke();
```

Comment:
You might be able to get by just checking for 'budget-inspector' as you were previously doing. Most features start out checking for the minimal changed nodes and then check for more only if they have to. It all depends on what the feature is doing.